File: ShopAppSpring/src/main/java/com/duchung/shopappspring/dtos/ExportRawProductDTO.java
package com.duchung.shopappspring.dtos;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class ExportRawProductDTO {

    private Long id;

    private String name;

    private double price;

    private Long quantity;

    private String note;

    @JsonProperty("store_id")
    private Long storeId;
}

File: shopapp-angular/src/app/manager/dtos/export.raw.product.dto.ts
export class ExportRawProductDTO {
  id: number;
  name: string;
  price: number;
  quantity: number;
  store_id: number;

    constructor(data: any) {
        this.id = data.id;
    this.name = data.name;
    this.price = data.price;
    this.quantity = data.quantity;
    this.store_id = data.store_id;
  }
}